### DISCUSSION TITLE
Emoji picker doesn't always render

### DISCUSSION THREAD

**Discussion by TwinFeats** [joeattardi/picmo#257] [Q&A]
I'm including this as a simple <script> tag from the UMD, and sometimes the emoji picker renders fine, sometimes only with the placeholders. The split of working/failing is about 10%/90%. No errors in the JS console. Any ideas?

**Comment by joeattardi** [comment]
Please create a CodePen/CodeSandbox reproducing the issue and I can look into this for you.

**Reply by TwinFeats** [reply]
Last night I figured what was really happening, it wasn't this Emoji library it was Vue.js getting in the way with a timing issue. Once I moved the container for the emojis outside of the Vue App container everything was fine. Thanks for the offer, though!
